scvi/model/_totalvi.py

```diff
@@ -1249,3 +1260,66 @@ def setup_anndata(
         )
         adata_manager.register_fields(adata, **kwargs)
         cls.register_manager(adata_manager)
+
+    @classmethod
+    def setup_mudata(
+        cls,
+        mdata: MuData,
+        layer_mod: Optional[str],
+        protein_expression_mod: str,
+        layer: Optional[str] = None,
```

Comment:
I think we can rename so as to change `protein_expression` -> `protein` and `layer` and `layer_mod` to have an `rna` prefix